administrator/components/com_scheduler/src/View/Task/HtmlView.php

```diff
@@ -0,0 +1,166 @@
+	 *
+	 * @var  CMSObject
+	 * @since  __DEPLOY__VERSION__
+	 */
+	protected $canDo;
+
+	/**
+	 * Overloads the parent constructor.
+	 * Just needed to fetch the Application object.
+	 *
+	 * @param   array  $config  A named configuration array for object construction.
+	 *                          name: the name (optional) of the view (defaults to the view class name suffix).
+	 *                          charset: the character set to use for display
+	 *                          escape: the name (optional) of the function to use for escaping strings
+	 *                          base_path: the parent path (optional) of the views directory (defaults to the component folder)
+	 *                          template_plath: the path (optional) of the layout directory (defaults to base_path + /views/ + view name
+	 *                          helper_path: the path (optional) of the helper files (defaults to base_path + /helpers/)
+	 *                          layout: the layout (optional) to use to display the view
+	 *
+	 * @throws Exception
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public function __construct($config = array())
+	{
+		$this->app = Factory::getApplication();
+		parent::__construct($config);
+	}
+
+	/**
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return void
+	 *
+	 * @throws Exception
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public function display($tpl = null): void
+	{
+		/*
+		 * Will call the getForm() method of TaskModel
+		 */
+		$this->form = $this->get('Form');
+		$this->item = $this->get('Item');
+		$this->state = $this->get('State');
+		$this->canDo = ContentHelper::getActions('com_scheduler', 'task', $this->item->id);
+
+		$this->addToolbar();
+		parent::display($tpl);
+	}
+
+	/**
+	 * Adds the page title and toolbar
+	 *
+	 * @return void
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected function addToolbar(): void
+	{
+		$app = $this->app;
+
+		$app->getInput()->set('hidemainmenu', true);
+		$user = $app->getIdentity();
+		$userId = $user->id;
+		$isNew = ($this->item->id == 0);
+		$canDo = $this->canDo;
+
+		ToolbarHelper::title($isNew ? Text::_('COM_SCHEDULER_MANAGER_TASK_NEW') : Text::_('COM_SCHEDULER_MANAGER_TASK_EDIT'), 'clock');
+
+		// Goes into ToolbarHelper::saveGroup()
+		$toolbarButtons = [];
+
+		// For a new task, check if user has 'core.create' access
+		if ($isNew && $canDo->get('core.create'))
+		{
+			// The task.apply task maps to the save() method in TaskController
+			ToolbarHelper::apply('task.apply');
+
+			$toolbarButtons[] = ['save', 'task.save'];
+		}
+		else
+		{
+			if (!$isNew && $canDo->get('core.edit'))
+			{
+				ToolbarHelper::apply('task.apply');
+				$toolbarButtons[] = ['save', 'task.save'];
+
+				// @todo | ? : Do we need save2new and save2copy? If yes, need to support in the Model,
+				// 			  here and the Controller.
+			}
+		}
+
+		ToolbarHelper::saveGroup(
```

Comment:
What if ```$toolbarButtons``` is empty?

Reply:
I've updated the toolbar and replaced the group with a simple save since we don't support save2copy etc.